<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>voltagepage</class>
 <widget class="QWidget" name="voltagepage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="VBox1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>224</width>
     <height>392</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="V1">
      <property name="title">
       <string>V1</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>   电压</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcd_v1"/>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>mV</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="V2">
      <property name="title">
       <string>V2</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>   电压</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcd_v2"/>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>mV</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="V3">
      <property name="title">
       <string>V3</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>   电压</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcd_v3"/>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>mV</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="V4">
      <property name="title">
       <string>V4</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>   电压</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcd_v4"/>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>mV</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="V5">
      <property name="title">
       <string>V5</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>   电压</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcd_v5"/>
       </item>
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>mV</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="V6">
      <property name="title">
       <string>V6</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>   电压</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcd_v6"/>
       </item>
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>mV</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="VBox2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>20</y>
     <width>224</width>
     <height>392</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGroupBox" name="V7">
      <property name="title">
       <string>V7</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>   电压</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcd_v7"/>
       </item>
       <item>
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>mV</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="V8">
      <property name="title">
       <string>V8</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>   电压</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcd_v8"/>
       </item>
       <item>
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>mV</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="V9">
      <property name="title">
       <string>V9</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>   电压</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcd_v9"/>
       </item>
       <item>
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>mV</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="V10">
      <property name="title">
       <string>V10</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>   电压</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcd_v10"/>
       </item>
       <item>
        <widget class="QLabel" name="label_20">
         <property name="text">
          <string>mV</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="V11">
      <property name="title">
       <string>V11</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <widget class="QLabel" name="label_21">
         <property name="text">
          <string>   电压</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcd_v11"/>
       </item>
       <item>
        <widget class="QLabel" name="label_22">
         <property name="text">
          <string>mV</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="V12">
      <property name="title">
       <string>V12</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <item>
        <widget class="QLabel" name="label_23">
         <property name="text">
          <string>   电压</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcd_v12"/>
       </item>
       <item>
        <widget class="QLabel" name="label_24">
         <property name="text">
          <string>mV</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
